README.md

```diff
@@ -142,11 +143,16 @@ ytdl cannot download videos that fall into the following
 * Rentals (if you have access, requires [cookies](example/cookies.js))
 
 Generated download links are valid for 6 hours, and may only be downloadable from the same IP address.
+Using IPv6 should not cause same IP address limitation.
 
 ## Handling Separate Streams
 
 Typically 1080p or better video does not have audio encoded with it. The audio must be downloaded separately and merged via an appropriate encoding library. `ffmpeg` is the most widely used tool, with many [Node.js modules available](https://www.npmjs.com/search?q=ffmpeg). Use the `format` objects returned from `ytdl.getInfo` to download specific streams to combine to fit your needs. Look at [example/ffmpeg.js](example/ffmpeg.js) for an example on doing this.
 
+## How does using an IPv6 block help?
+
+Using an IPv6 block is essentially using millions of different IPv6 addresses. Changing the external IP that you use is similar to proxying except without bypassing any restrictions such as a region lock. In a /64 IPv6 block, there are 18,446,744,073,709,551,616 single IPv6 addresses. This would allow you to have many requests without using the same IPv6 address more than once.
```

Comment:
```suggestion
Using an IPv6 block is essentially having millions of IPv6 addresses at your request. Changing the external IP that you use is similar to using a proxy, except without bypassing restrictions such as a region lock. In a /64 IPv6 block, there are 18,446,744,073,709,551,616 unique IPv6 addresses. This would allow you to make each request with a different IPv6 address.
```